zerver/openapi/zulip.yaml

```diff
@@ -18608,9 +18614,46 @@ paths:
                             type: integer
                           example: [12, 13]
                   example: 12
+                can_mention_group_previous:
+                  description: |
+                    The current User group whose members are allowed to
+                    mention the group.
+
+                    This can either be ID of a named user group or an object
+                    containing direct members and direct subgroups of an
+                    anonymous user group.
+
+                    The request will be rejected if this value does not match
+                    the actual setting value.
```

Comment:
This should explain the purpose of this parameter a bit more clearly -- the fact that it's optional, but also that the purpose is to prevent races by having the client send what value it expects the server to already have.

I guess that does present a question of whether we want to call this `_expected_current` instead of `_previous`; probably worth discussing that detail a bit.